### ISSUE TITLE
Tesla Troopers (debugin)

### ISSUE THREAD

**Issue by xplayer51** [OpenRA/OpenRA#13392]
Hiya, Long time C&C fan here... while doing some skirmish mode fun I was just playing with the different unit types, and I went to train some Tesla troopers, which wouldn't be a issue.... but when I go to click on the unit before he says (Insert Arnold like voice clip here) the game crashes to windows and gives me a fatal error message. I don't know what this issue is as I've checked multiple times (even checked the error logs, but I couldn't find anything?) and still crashes the game. Just hope this message helps to get it noticed and fixed. I know it's a Aftermath unit but still a good unit, but one crashes the game, that's a big deal. I'm starting to like this program It has really improved since older versions, as obviously running old RA95 on win7 or 10 just ain't gonna happen otherwise, and I like the Tiberian sun\ RA2 style mechanics IE. flying rockets and projectiles, better resolution scaling + optimization for newer hardware & even still being able to install data from the old Cd's is great. So just I hope I'd might help out with this "Fatal" in-game bug I found. C:

Thank you,
B.    Xplayer51

**Comment by rob-v** [comment]
I am not sure. Which mod, version do you mean? After crash there is no debug/exception file in \Documents\OpenRA\Logs (on Win)?

**Comment by xplayer51** [comment]
Well for the "mod version" I'm using the latest build of OpenRA, and I've used my old CD-rom copy of Aftermath for the files. I have ran the aftermath expansion CD on a older system and the shock trooper works fine. But I did look into the last log just after checking here to see what the solution might be and I think I found the problem in the last log. (The site won't let me attach the text file for some reason, here is the raw bit. I will bold the issue I found.)
_______________________________________________________
Red Alert Mod at Version release-20170421
on map 58da5c480d13a00a07c9bee00f928f70564f188f (Nine Lives by PizzaAtomica).
Date: 2017-05-30 01:55:14Z
Operating System: Windows (Microsoft Windows NT 6.1.7601 Service Pack 1)
Runtime Version: .NET CLR 4.0.30319.42000
**## Exception of type `System.IO.InvalidDataException`: jjump1.aud is not a valid sound file!**
   at OpenRA.Sound.LoadSound(ISoundLoader[] loaders, IReadOnlyFileSystem fileSystem, String filename)
   at OpenRA.Sound.<Initialize>c__AnonStorey0.<>m__0(String s)
   at OpenRA.Exts.GetOrAdd[K,V](Dictionary`2 d, K k, Func`2 createFn)
   at OpenRA.Primitives.Cache`2.get_Item(T key)
   at OpenRA.Sound.PlayPredefined(SoundType soundType, Ruleset ruleset, Player p, Actor voicedActor, String type, String definition, String variant, Boolean relative, WPos pos, Single volumeModifier, Boolean attenuateVolume)
   at OpenRA.Mods.Common.Traits.Voiced.PlayVoice(Actor self, String phrase, String variant)
   at OpenRA.VoiceExts.PlayVoice(Actor self, String phrase)
   at OpenRA.Selection.Combine(World world, IEnumerable`1 newSelection, Boolean isCombine, Boolean isClick)
   at OpenRA.Mods.Common.Widgets.WorldInteractionControllerWidget.HandleMouseInput(MouseInput mi)
   at OpenRA.Widgets.Widget.HandleMouseInputOuter(MouseInput mi)
   at OpenRA.Widgets.Ui.HandleInput(MouseInput mi)
   at OpenRA.DefaultInputHandler.<OnMouseInput>c__AnonStorey2.<>m__0()
   at OpenRA.Sync.CheckSyncUnchanged[T](World world, Func`1 fn)
   at OpenRA.DefaultInputHandler.OnMouseInput(MouseInput input)
   at OpenRA.Platforms.Default.Sdl2Input.PumpInput(Sdl2GraphicsDevice device, IInputHandler inputHandler)
   at OpenRA.Platforms.Default.Sdl2GraphicsDevice.PumpInput(IInputHandler inputHandler)
   at OpenRA.Renderer.EndFrame(IInputHandler inputHandler)
   at OpenRA.Game.RenderTick()
   at OpenRA.Game.Loop()
   at OpenRA.Game.Run()
   at OpenRA.Program.Run(String[] args)
